### DISCUSSION TITLE
ext4 file system support

### DISCUSSION THREAD

**Discussion by shaggygi** [zephyrproject-rtos/zephyr#66292] [Q&A]
Just curious as I have not seen any results searching the web, but does Zephyr support ext4 file system?  For example, if you wanted to interact with files (create/copy/read/write/delete) with a connected eMMC, etc.

Any info/links will be appreciated.

**Comment by nordicjm** [comment]
Very limited ext2 support, depending upon block size, up to a 128MB partition.
